tests/integration/deliver_grant_funding/routes/test_reports.py

```diff
@@ -956,6 +1027,45 @@ def test_post(self, authenticated_grant_admin_client, factories, db_session):
         assert get_h1_text(soup) == "Edit question"
         assert get_h2_text(soup) == "Question created"
 
+    def test_post_add_to_group(self, authenticated_grant_admin_client, factories, db_session):
+        grant = authenticated_grant_admin_client.grant
+        report = factories.collection.create(grant=grant, name="Test Report")
+        db_form = factories.form.create(section=report.sections[0], title="Organisation information")
+        group = factories.group.create(form=db_form, name="Test group", order=0)
+
+        form = QuestionForm(
+            data={
+                "text": "question",
+                "hint": "hint text",
+                "name": "question name",
+            },
+            question_type=QuestionDataType.TEXT_SINGLE_LINE,
+        )
+        response = authenticated_grant_admin_client.post(
+            url_for(
+                "deliver_grant_funding.add_question",
+                grant_id=grant.id,
+                form_id=db_form.id,
+                question_type=QuestionDataType.TEXT_SINGLE_LINE.name,
+                parent_id=group.id,
+            ),
+            data=form.data,
+            follow_redirects=False,
+        )
+        assert response.status_code == 302
+        assert response.location == AnyStringMatching("/grant/[a-z0-9-]{36}/question/[a-z0-9-]{36}")
```

Comment:
optional nit: Not totally sure about doing this as the main part of the test, I feel like under `TestListTaskQuestions` the only endpoints called should be that one. This feels like more of a test of the `AddQuestion` endpoint? And then the assertions after getting the redirect could be stripped out. It just feels a bit of a conflated test to me.